SPECS/ltrace/ltrace.spec

```
@@ -42,16 +42,16 @@ make install DESTDIR=%{buildroot}
 %check
 make %{?_smp_mflags} check
 
-%clean
-rm -rf %{buildroot}/*
-
 %files
 %defattr(-,root,root)
 %license COPYING
 %{_sysconfdir}/ltrace.conf
 %{_bindir}/*
 %{_datadir}
 %changelog
```

Maybe we should add a new line while we are here:
```suggestion

%changelog
```

---

Sure.